src/BulletGauge/BulletGauge.tsx

```diff
@@ -0,0 +1,78 @@
+import { BulletGaugeProps } from "./BulletGauge.types";
```

Comment:
I think you are missing the ClientOnly wrapper to support SSR. Refer to the SurfacePlot as an example.

https://github.com/IPG-Automotive-UK/react-ui/blob/main/src/SurfacePlot/SurfacePlotClientOnly.tsx#L1-L19

https://github.com/IPG-Automotive-UK/react-ui/blob/0cc95cce1e3324582cebe767e766b77ffbc9b1ce/src/SurfacePlot/index.ts#L4